test/serdes_test.c

```diff
@@ -15,11 +15,22 @@
 #include <openssl/serializer.h>
 #include <openssl/deserializer.h>
 
+#include "internal/cryptlib.h"   /* ossl_assert */
+
 #include "testutil.h"
 
+/*
+ * TODO(3.0) Modify PEM_write_bio_PrivateKey_traditional() to handle
```

Comment:
Is this something you plan on adding as a seperate PR?

Reply:
Separate PR, yes.  Me, or someone else...  we'll have quite a few TODOs like this to go through.